resources/conf/config.toml.template

```diff
@@ -492,6 +492,10 @@ enabled = true
   # Enforcer related configurations
   [analytics.enforcer]
     [analytics.enforcer.configProperties]
+      # Enables elk analytics logging
+      type = "elk"
```

Comment:
Shall we move this type under analytics root section? No need to move this type thing in analytics.enforcer.configProperties

Reply:
+1